russian/zefir.md

```diff
@@ -53,3 +53,5 @@
 [Почесать в подмышечной впадине.](scratch/scratch.md)
 
 [Пойду в другую комнату](next_room/next_room.md)
+
+[Поговорю с соседями](speak_to_neighbor)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.